packages/bpk-component-drawer/readme.md

```diff
@@ -46,13 +46,15 @@ class App extends Component {
           <BpkButton onClick={this.onOpen}>Open drawer</BpkButton>
         </div>
         <BpkDrawer
+          id="my-drawer"
           isOpen={this.state.isOpen}
           onClose={this.onClose}
           title="Drawer title"
```

Comment:
`title` is required unless you specify `hideTitle` right?

Reply:
It's required all the time. `hideTitle` visually hides the title but it needs to be there for a11y.

Reply:
Neat :)